RELEASE/scripts/autoscend.ash

```diff
@@ -6407,9 +6433,14 @@ boolean L11_unlockHiddenCity()
 	visit_url("choice.php?whichchoice=125&option=3&pwd");
 	auto_log_info("Hidden Temple Unlocked");
 	set_property("auto_hiddenunlock", "finished");
-	if(!possessEquipment($item[Antique Machete]) && !in_hardcore())
+	if(((!possessEquipment($item[Antique Machete])) || (!possessEquipment($item[Muculent Machete]))) && !in_hardcore())
```

Comment:
you have way more paren's here than you need.

Reply:
Ok, Fixed.

That is a personal foible, I tend to put each test in its own parenthesis/bracket.